File: rails-demo/app/controllers/tables_controller.rb
class TablesController < ApplicationController
  def bootstrap_sortable
    @title = 'Bootstrap Sortable - Tables'
  end

  def bootstrap_table
    @title = 'Bootstrap table - Tables'
  end

  def bootstrap
    @title = 'Bootstrap - Tables'
  end

  def datatables
    @title = 'DataTables - Tables'
  end

end

File: laravel-demo/app/Http/Controllers/TablesController.php
<?php

namespace App\Http\Controllers;

use App\User;
use App\Http\Controllers\Controller;

class TablesController extends Controller
{
    public function bootstrapSortable()
    {
        return view('tables.bootstrap-sortable', ['title' => 'Bootstrap Sortable - Tables']);
    }

    public function bootstrapTable()
    {
        return view('tables.bootstrap-table', ['title' => 'Bootstrap table - Tables']);
    }

    public function bootstrap()
    {
        return view('tables.bootstrap', ['title' => 'Bootstrap - Tables']);
    }

    public function datatables()
    {
        return view('tables.datatables', ['title' => 'DataTables - Tables']);
    }

}
